biodatasets/scifact/scifact.py

```diff
@@ -0,0 +1,439 @@
+  pages     = {7534--7550},
+  biburl    = {},
+  bibsource = {}
+}
+"""
+
+_DATASETNAME = "scifact"
+
+_SOURCE_CORPUS_DESCRIPTION = """\
+    SciFact is a dataset of 1.4K expert-written scientific claims paired with evidence-containing abstracts, and annotated with labels and rationales.
+    This config has abstracts and document ids.
+    """
+
+_SOURCE_CLAIMS_DESCRIPTION = """\
+    SciFact is a dataset of 1.4K expert-written scientific claims paired with evidence-containing abstracts, and annotated with labels and rationales.
+    This config connects the claims to the evidence and doc ids.
+    """
+
+_BIGBIO_ENTAILMENT_RATIONALE_DESCRIPTION = """\
+    SciFact is a dataset of 1.4K expert-written scientific claims paired with evidence-containing abstracts, and annotated with labels and rationales.
+    This task is the following: given a claim and an abstract, label that sentence with a rationale/not_rationale indicating if it is evidence (can be supporting or refuting). This corresponds to the second task outlined in Section 5 of the paper."
+    """
+
+_BIGBIO_ENTAILMENT_LABELPREDICTION_DESCRIPTION = """\
+    SciFact is a dataset of 1.4K expert-written scientific claims paired with evidence-containing abstracts, and annotated with labels and rationales.
+    This task is the following: given a claim and several sentences of evidence (that either support or refute the claim), label that sentence with one of {REFUTES, SUPPORTS, NOINFO}. This corresponds to the third task outlined in Section 5 of the paper.
+    """
+
+_DESCRIPTION = {
+    "scifactcorpus_source": _SOURCE_CORPUS_DESCRIPTION,
+    "scifactclaims_source": _SOURCE_CLAIMS_DESCRIPTION,
+    "scifactrationale_bigbio_entailment": _BIGBIO_ENTAILMENT_RATIONALE_DESCRIPTION,
+    "scifactlabelprediction_bigbio_entailment": _BIGBIO_ENTAILMENT_LABELPREDICTION_DESCRIPTION,
+}
+
+_HOMEPAGE = "https://scifact.apps.allenai.org/"
+
+
+_LICENSE = "CC BY-NC 2.0"
+
+_URLS = {
+    _DATASETNAME: "https://scifact.s3-us-west-2.amazonaws.com/release/latest/data.tar.gz",
+}
+
+_SUPPORTED_TASKS = [Tasks.TEXTUAL_ENTAILMENT]
+
+_SOURCE_VERSION = "1.0.0"
+
+_BIGBIO_VERSION = "1.0.0"
+
+
+@dataclass
+class BigBioConfig(datasets.BuilderConfig):
+    """BuilderConfig for BigBio."""
+
+    name: str = None
+    version: str = None
+    description: str = None
+    schema: str = None
+    subset_id: str = None
+
+
+class SciFact(datasets.GeneratorBasedBuilder):
+    """
+    SciFact is a dataset of 1.4K expert-written scientific claims paired with evidence-containing abstracts, and annotated with labels and rationales.
+    """
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    BIGBIO_VERSION = datasets.Version(_BIGBIO_VERSION)
+
+    BUILDER_CONFIGS = [
+        BigBioConfig(
+            name="scifactcorpus_source",
+            version=SOURCE_VERSION,
+            description="scifact source schema for the corpus config",
+            schema="source",
+            subset_id="scifact",
```

Comment:
can we use the pattern, 
`name = scifact_corpus_source` 
`subset_id = scifact_corpus` 
for the config names and subset_ids please.